### PR TITLE
Feature/create vrules

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/tools/rules_utils.py [hunk 1]
```diff
@@ -335,3 +590,29 @@ def validate_all_rules(self):
...
+        collection_dict = driver.current_collection(CEP_MONGO_RULES_COLLECTION)
+        cursor =  driver.find_data(collection_dict, {NAME: RULE_CARD_DICT[NAME]})
+        for doc in cursor:
+            temp_name = doc[NAME]
+        assert temp_name == RULE_CARD_DICT[NAME], "The rule %s has not been created..." % (RULE_CARD_DICT[NAME])
+
+
+
+
+    # """
```
Comment (jcalderin) [inline test/acceptance/tools/rules_utils.py]: This comments can be deleted safetly?

File: test/acceptance/integration/Rules/Cards/card_create_rules.feature [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+     # Time cards
+    And create a time card of time elapsed type, with id "card_10", interval "8" and connect to "card_11"
+     # end cards
+    When append a new rule name "<rule_name>", activate "1"
+    Then I receive an "Created" http code
+    And Validate that rule name is created successfully in db
+    And delete a rule created
+  Examples:
+    | rule_name |
+    | test_1    |
```
Comment (jcalderin) [inline test/acceptance/integration/Rules/Cards/card_create_rules.feature]: Why unnecessary Scenario Outline

File: test/acceptance/tools/rules_utils.py (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/tools/rules_utils.py]: Fixed in 84aea9d

File: test/acceptance/integration/Rules/Cards/card_create_rules.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/Cards/card_create_rules.feature]: In some cases are not necessary the Scenario outline

File: test/acceptance/__init__.py [hunk 3]
```diff
@@ -1 +1,25 @@
...
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
+
```
Comment (fgalan) [inline test/acceptance/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/integration/Rules/Cards/__init__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
```
Comment (fgalan) [inline test/acceptance/integration/Rules/Cards/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/integration/Rules/EPL/__init__.py [hunk 5]
```diff
@@ -1 +1,25 @@
...
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
+
```
Comment (fgalan) [inline test/acceptance/integration/Rules/EPL/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/integration/__init__.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
```
Comment (fgalan) [inline test/acceptance/integration/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/integration/notifications/EPL/__init__.py [hunk 7]
```diff
@@ -1 +1,25 @@
...
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
+
```
Comment (fgalan) [inline test/acceptance/integration/notifications/EPL/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/integration/notifications/__init__.py [hunk 8]
```diff
@@ -1 +1,25 @@
...
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with perseo-fe.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with:
+#   iot_support at tid.es
+#
+
```
Comment (fgalan) [inline test/acceptance/integration/notifications/__init__.py]: This file only contatins the commented header? Mabye the `__author__` line is missing at the bottom...

File: test/acceptance/tools/rules_utils.py [hunk 9]
```diff
@@ -79,7 +79,78 @@
...
+USER_PARAMS          = u'userParams'
+ACTIVE               = u'active'
+CARDS                = u'cards'
+X_AUTH_TOKEN         = u'X-Auth-Token'
+X_AUTH_TOKEN_VALUE   = u'MCAQExCTAHBgUrDgMCGjCCAlwGCSqGSIb3DQEHAaCCAk0EggJJeyJ0b2tlbiI6IHsibWV0aG9kcyI6IFsidG9rZW4iLCAicGFzc3dvcmQiXSwgInJvbGVzIjogW3siaWQiOiAiMjkxOWYzZTRjYTJlNGI3Mjg0OWJmZDAxZGIwZThiMWYiLCAibmFtZSI6ICJkNGNkNjcwOTNmMWQ0Y2U4OWNiODUwYWU3MWQ4NDgxNyNTdWJTZXJ2aWNlQWRtaW4ifV0sICJleHBpcmVzX2F0IjogIjIwMTUtMDItMDRUMTA6Mzg6NDcuODEyOTk1WiIsICJwcm9qZWN0IjogeyJkb21haW4iOiB7ImlkIjogImQ0Y2Q2NzA5M2YxZDRjZTg5Y2I4NTBhZTcxZDg0ODE3IiwgIm5hbWUiOiAiU21hcnRWYWxlbmNpYSJ9LCAiaWQiOiAiMjVkM2ViZDljMWIxNDE3ZDk5NGI5MGYzN2NmM2YxYmUiLCAibmFtZSI6ICIvRWxlY3RyaWNpZGFkIn0sICJjYXRhbG9nIjogW10sICJleHRyYXMiOiB7fSwgInVzZXIiOiB7ImRvbWFpbiI6IHsiaWQiOiAiZDRjZDY3MDkzZjFkNGNlODljYjg1MGFlNzFkODQ4MTciLCAibmFtZSI6ICJTbWFydFZhbGVuY2lhIn0sICJpZCI6ICJkOWQ1NGVjMjE1NjM0OWI4OTRlNTVmMjE2NWM1MmQ4MyIsICJuYW1lIjogIkFsaWNlIn0sICJpc3N1ZWRfYXQiOiAiMjAxNS0wMi0wNFQwOTozODo0OS4zMzM2MDBaIn19MYIBgTCCAX0CAQEwXDBXMQswCQYDVQQGEwJVUzEOMAwGA1UECAwFVW5zZXQxDjAMBgNVBAcMBVVuc2V0MQ4wDAYDVQQKDAVVbnNldDEYMBYGA1UEAwwPd3d3LmV4YW1wbGUuY29tAgEBMAcGBSsOAwIaMA0GCSqGSIb3DQEBAQUABIIBAJmYKdlwuXALebkqt'
```
Comment (fgalan) [inline test/acceptance/tools/rules_utils.py]: I understand this code is not actually valid, it's only for tests (just to be sure... we have had some bad experience in the past about things like this :)

File: test/acceptance/tools/rules_utils.py (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/tools/rules_utils.py]: Fixed in 84aea9d

File: test/acceptance/integration/Rules/Cards/card_create_rules.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/Cards/card_create_rules.feature]: FIXED in f6e56c1

File: test/acceptance/integration/__init__.py (hunk 6 above)
Comment (iariasleon) [inline test/acceptance/integration/__init__.py]: FIXED in f6e56c1

File: test/acceptance/integration/notifications/EPL/__init__.py (hunk 7 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/EPL/__init__.py]: FIXED in f6e56c1

File: test/acceptance/integration/notifications/__init__.py (hunk 8 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/__init__.py]: FIXED in f6e56c1

File: test/acceptance/tools/rules_utils.py (hunk 9 above)
Comment (iariasleon) [inline test/acceptance/tools/rules_utils.py]: That´s right, this code is not valid, but to try to simulate a request from portal, is necessary a value. Changed to fake value.

Comment (iariasleon) [inline test/acceptance/tools/rules_utils.py]: Fixed in 4d147b4

File: test/acceptance/integration/Rules/EPL/__init__.py (hunk 5 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/EPL/__init__.py]: FIXED in f6e56c1

File: test/acceptance/integration/Rules/Cards/__init__.py (hunk 4 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/Cards/__init__.py]: FIXED in f6e56c1

**Comment by jcalderin** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**